### DISCUSSION TITLE
Can't differentiate alphabet F & G?

### DISCUSSION THREAD

**Discussion by iamdhruvanand** [GorvGoyl/ChatGPT-Writer#87] [Report Bugs and Issues]
This is what I asked ChatGPT:
Name the countries that start with alphabet F

This is the response I got:
Sure, here is a list of countries that start with the letter "F":

Fiji
Finland
France
Gabon
Gambia
Georgia
Germany
Ghana
Greece
Grenada

**Comment by GorvGoyl** [comment]
it gave right answer to me
<img width="770" alt="screenshot 2023-04-13 at 21 38 35@2x" src="https://user-images.githubusercontent.com/7106086/231794813-c3fe32cd-9960-482d-9d2c-6744b1585655.png">

<img width="770" alt="screenshot 2023-04-13 at 21 38 35@2x" src="https://user-images.githubusercontent.com/7106086/231794701-6ebea8de-850a-4ec4-ac0a-a5350be1b85d.png">

**Reply by iamdhruvanand** [reply]
To me now it still doesn't show the correct answer:
![image](https://user-images.githubusercontent.com/48094363/231879417-f137c285-6957-4ce8-b1bb-58d5a4ae2b8f.png)

**Reply by iamdhruvanand** [reply]
This is the one from before:
![image](https://user-images.githubusercontent.com/48094363/231879588-1b8c348a-3802-4d3b-9905-70c28714a5b9.png)

**Reply by iamdhruvanand** [reply]
Tried to ask in a different way:
![image](https://user-images.githubusercontent.com/48094363/231881072-21d3f050-1a61-4799-88ad-189321eeed3b.png)
